src/components/form.js

```diff
@@ -1,31 +1,45 @@
 import React, { useState } from 'react';
 import { useDispatch } from 'react-redux';
-import { nanoid } from 'nanoid';
-import { addBooks } from '../redux/Books/books';
+import { v4 as uuidv4 } from 'uuid';
+import { addBook } from '../redux/Books/books';
 
 function Forms() {
-  const [title, setTitle] = useState('');
-  const [author, setAuthor] = useState('');
+  const [cue, setCue] = useState({
+    title: '',
+    author: '',
+  });
+
   const dispatch = useDispatch();
+
+  const manageInfo = (e) => {
+    const input = e.target.value;
+    switch (e.target.id) {
+      case 'title':
+        setCue((inputs) => ({ ...inputs, title: input }));
+        break;
+      default:
+        break;
+    }
+  };
+
   const handSubmit = (e) => {
     e.preventDefault();
-    const book = {
-      id: nanoid(),
-      title,
-      author,
+    const newBook = {
+      item_id: uuidv4(),
+      title: cue.title,
+      author: cue.author,
+      category: 'none',
     };
-    setTitle('');
-    setAuthor('');
-    dispatch(addBooks(book));
+    dispatch(addBook(newBook));
   };
 
   return (
     <div className="genaral-contain">
       <h1 className="heading"> ADD NEW BOOK</h1>
       <form className="list-form" onSubmit={handSubmit}>
-        <input type="text" name="title" placeholder="Book Title Here" value={title} onChange={(e) => setTitle(e.target.value)} />
-        <input type="text" name="author" placeholder="Author Name Here" value={author} onChange={(e) => setTitle(e.target.value)} />
-        <button className="button-sub" type="submit">Add Book</button>
+        <input type="text" name="title" placeholder="Book Title Here" onChange={manageInfo} value={cue.title} />
+        <input type="text" name="author" placeholder="Author Name Here" onChange={manageInfo} value={cue.author} />
```

Comment:
- [ ] Kindly make sure that after refactoring your code to use API, the add book and remove book features should still work as before. Right now, I can't even input text into these fields to add a book.